### DISCUSSION TITLE
Automated Location Fetch Toggle

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#112] [Ideas]
An optional toggle for automated location fetching. 

Fetching can be on the open app or MS timer, or even a toggle for both.

**Comment by farfromrefug** [comment]
@jacobgonzales20 not sure about this one. My weather app was specifically created in the first place to not do anything in the background and use as less  bandwith/cpu as possible. Having things running without the user knowledge is something i would prefer to avoid.
Plus most users wont change location that often so clicking a menu or something does not seem like a big deal to me.
But as always i am open to discussion

**Reply by ghost** [reply]
I'm glad you are holding to those values, and I am looking forward to seeing where this app goes.

**Reply by farfromrefug** [reply]
@jacobgonzales20 thanks! we LL see indeed ;) 
BTW you are an iOS user right ?

**Reply by ghost** [reply]
> @jacobgonzales20 thanks! we LL see indeed ;) 
> BTW you are an iOS user right ?

Correct. iPhone 14 Pro on iOS 17.3.1

**Reply by farfromrefug** [reply]
@jacobgonzales20 OK in might need your advice. Apple is now refusing my apps submission because I show a donation banner and they want me to use in app purchase to get 30%... 
I am not sure what to do:
* use in app purchase for donations(dont really like the idea of going through apple and giving 30% of people Money to them)
* put a link to github only and hoping people will see the sponsoring on there.

I am really not an iOS use anymore so I am not sure what to do when it is related to OSS on there. your opinion would be valuable

**Reply by ghost** [reply]
This is one of Apple's biggest downsides. I would say comply now as with the EU 3rd party app store changes, I would give it a year till it comes to the US. From a marketing perspective as this is what I specialize in. Playing the long game for this control Apple demands might be the best to stay on the app store and grow a following for the app.

I will say Yattee, a youtube frontend, has some experience with Apple. As they have a youtube frontend on the app store which is insane. I would check out their github

https://github.com/yattee/yattee
